src/etc/ssl/opnsense.cnf

```diff
@@ -229,7 +231,7 @@ authorityKeyIdentifier=keyid:always
 basicConstraints=CA:FALSE
 
 # This will be displayed in Netscape's comment listbox.
-nsComment			= "OpenSSL Generated Certificate"
+nsComment			= "OPNsense Generated Certificate Revocation List"
```

Comment:
this is under [ proxy_cert_ext ], not [ crl_ext ] -- is this correct?

Reply:
You are absolutely right! I didn't notice [ proxy_cert_ext ] at all. Fixed it in the next commit.